### ISSUE TITLE
Default gas amount

### ISSUE THREAD

**Issue by okwme** [samepant/cosmos-multisig-ui#1]
Successfully crafted and signed transaction to send `1uatom` but tx failed to submit because of gas check(?). Not sure why it would fail tho, fees are 0 so gas shouldn't matter, maybe it needs to be some randomly high number that doesn't matter because it will ultimately be 0 anyway... 
```
height: 0
txhash: 7B476305A20CCE21ABD0C57B3E94C9E63800A3105708E085F6E1113869B73BB3
code: 12
data: ""
rawlog: '{"codespace":"sdk","code":12,"message":"out of gas in location: txSize; gasWanted:
  2000, gasUsed: 3060"}'
logs: []
info: ""
gaswanted: 0
gasused: 0
codespace: ""
tx: null
timestamp: ""
events: []
```

**Comment by okwme** [comment]
changing to 20000 instead of 2000 made it successful to submit altho it still failed during execution because it asked to use 22304 gas instead of the allotted 20000.
```
  "raw_log": "{\"codespace\":\"sdk\",\"code\":12,\"message\":\"out of gas in location: WritePerByte; gasWanted: 20000, gasUsed: 22304\"}",
```

Trying again with 25000 and get this error:
```
  "raw_log": "{\"codespace\":\"sdk\",\"code\":12,\"message\":\"out of gas in location: ReadFlat; gasWanted: 25000, gasUsed: 25574\"}",
```

Tried _one_ more time with 200000 and it worked 🎉

(nothing wrong with the code here, maybe worth adding a couple 0s to the default value but more of a problem with how the SDK does gas metering–an open issue)

**Comment by samepant** [comment]
Thanks for trying it out and the feedback 🌞

ah ok, I assumed the failure message was a result of the server having the `--dry-run` flag set when it runs `gaiacli tx broadcast` . It definitely feels like the gas portion of the app could be more sophisticated, either by estimating gas needed in the UI or surfacing more info on gas prices.

**Comment by samepant** [comment]
fixed in 861c3886fbae1659ae4e301c04be2de95fa171d8
